dist/tools/cc2538-bsl/Makefile

```diff
@@ -6,5 +6,10 @@ PKG_BUILDDIR=$(CURDIR)/bin
 
 include $(RIOTBASE)/pkg/pkg.mk
 
-all:
+all: $(CURDIR)/cc2538-bsl
+
+$(CURDIR)/cc2538-bsl:
 	cp $(PKG_SOURCE_DIR)/cc2538-bsl.py .
+
+clean::
+	rm -f $(CURDIR)/cc2538-bsl
```

Comment:
```suggestion
all: $(CURDIR)/cc2538-bsl.py

$(CURDIR)/cc2538-bsl.py:
	cp $(PKG_SOURCE_DIR)/cc2538-bsl.py .

clean::
	rm -f $(CURDIR)/cc2538-bsl.py
```

Reply:
I'm realizing this is not looking if there are new patches or patches differneces